_code-samples/nft-modular-tutorials/broker-nfts.js

```diff
@@ -0,0 +1,44 @@
+// *******************************************************
+// ******************* Broker Sale ***********************
+// *******************************************************
+
+async function brokerSale() {
+  const wallet = xrpl.Wallet.fromSeed(accountSeedField.value);
+  const net = getNet();
+  const client = new xrpl.Client(net);
+  let results = `\n=== Connected. Brokering the sale. ===`;
+  resultField.value = results;
+
+  try {
+    await client.connect();
+
+    // Prepare transaction -------------------------------------------------------
+    const brokerTx = {
+      "TransactionType": "NFTokenAcceptOffer",
+      "Account": wallet.classicAddress,
```

Comment:
I've noticed switching between using `classicAddress` and `address`. They both work, but for consistency and to avoid potential confusion that the two are different we should stick to one.